src/pageServices/connector.ts

```diff
@@ -37,10 +30,9 @@ class Connector {
     this.getTab();
   }
 
-  async initScripts(onInitHandler: () => Promise<void>, onDisconnectHandler: () => void) {
+  async initScripts() {
     await this.attachStaticScripts();
-    onInitHandler();
-    this.onTabUpdate(onInitHandler, onDisconnectHandler);
+    return "success";
   }
 
   handleError(error: Error) {
```

Comment:
I couldn't find usage of this method, can you pls check we need it?

Reply:
yes, we don't